internal/services/network/express_route_connection_resource.go

```diff
@@ -68,13 +68,31 @@ func resourceExpressRouteConnection() *pluginsdk.Resource {
 				Optional: true,
 			},
 
+			"enabled_express_route_gateway_bypass": {
```

Comment:
Can we rename this to
```suggestion
			"express_route_gateway_bypass_enabled": {
```